File: Java Programming/Competitive Programming/Coding Challenges 2/program3.java
/* 
3. Write a program to print all odd numbers up to N.
Input : 20
Output : Odd Numbers 1
         Odd Numbers 3
         Odd Numbers 5
         Odd Numbers 7
         Odd Numbers 9
         Odd Numbers 11
         Odd Numbers 13
         Odd Numbers 15
         Odd Numbers 17
         Odd Numbers 19
*/

class Logic
{
    void PrintOddNumbers (int iNo)
    {
        int iCnt =0;
        for( iCnt=0; iCnt <= iNo; iCnt++)
        {
            if (iCnt % 2==1) 
            {
                System.out.println("Odd Numbers "+iCnt);    
            }
        }
    }
}
class program3 
{
    public static void main(String[] args) 
    {
    Logic obj=new Logic();
    obj.PrintOddNumbers(20);
    }
}

File: C++ Programming/Competitive Programming/Coding Challenges 14/program3.cpp
/*
3. Write a program which accept range from user and return addition of all numbers in between that range. (Range should contains positive numbers only)
Input : 23 30
Output : 212
Input : 10 18
Output : 126
Input : -10 2
Output : Invalid range
Input : 90 18
Output : Invalid range
*/

#include <iostream>
using namespace std;

int RangeSum(int iStart, int iEnd)
{
    if ((iStart < 0) || (iEnd < 0) || (iStart > iEnd))
    {
        return -1;
    }

    int iSumStart = (iStart * (iStart - 1)) / 2;
    int iSumEnd = (iEnd * (iEnd + 1)) / 2;

    return iSumEnd - iSumStart;
}

int main()
{
    int iValue1 = 0;
    int iValue2 = 0;
    int iRet = 0;

    cout << "Enter starting point : ";
    cin >> iValue1;

    cout << "Enter ending point : ";
    cin >> iValue2;

    iRet = RangeSum(iValue1, iValue2);

    if (iRet == -1)
    {
        cout << "Invalid Range";
    }
    else
    {
        cout << "Addition is : " << iRet;
    }

    return 0;
}